src/RepublicanSilence/AdherentZone/CommitteeHostZoneRetriever.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace AppBundle\RepublicanSilence\AdherentZone;
+
+use AppBundle\Entity\Adherent;
+use AppBundle\Entity\CommitteeMembership;
+use Symfony\Component\HttpFoundation\Request;
+
+class CommitteeHostZoneRetriever implements AdherentZoneRetrieverInterface
```

Comment:
Retriever is a type of dog in English like in French.
IMO this is, either a `Manager` or a `Handler` or a `QueryHandler`.